### ISSUE TITLE
Is there synergy with SciSharp?

### ISSUE THREAD

**Issue by GeorgeS2019** [AMArostegui/SpacyDotNet#2]
Will there be advantage to be part of [SciSharp](https://github.com/SciSharp/SpaCy.NET/issues/2#issue-787749184)

**Comment by AMArostegui** [comment]
Nope. It looks like they aim to create a whole ecosystem, professional and production ready. This is a project of a solo developer with a limited scope.

**Comment by GeorgeS2019** [comment]
@AMArostegui   the .NET community needs SpaCy and by highlighting your project through SciSharp, more developers will join you to make SpaCy.Net more complete. 

GOOD job and it works.
